Parking_Lot_Reservation/Models/PersonModel.cs

```diff
@@ -13,8 +13,6 @@ public class PersonModel
         public int PersonId { get; set; }
         public string Name { get; set; }
         public string Surname { get; set; }
-
-        [ForeignKey("ParkingSpaceId")]
-        public virtual ParkingSpaceModel ParkingSpace { get; set; }
+        public virtual ICollection<ParkingSpaceModel> ParkingSpaceModels { get; set; }
```

Comment:
You can change name of this variable to something that instantly explains what it is for. Like "AssignesParkingSpaces"